sdk/cosmosdb/cosmos/test/public/integration/failover.spec.ts

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 import { CosmosClient, PluginOn, CosmosClientOptions, PluginConfig } from "../../../src";
+import { getEmptyCosmosDiagnostics } from "../../../src/CosmosDiagnostics";
```

Comment:
Why can't we just include in the above as its already imported from src folder

Reply:
`getEmptyCosmosDiagnostics` is an internal object, Not exposed as a type in `@azure/cosmos` package. It is a utility function getting reused in tests.